src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -36,46 +36,11 @@ const ActorInit En_Ginko_Man_InitVars = {
 };
 
 ActorAnimationEntry animations[] = {
-    {
-        object_ginko_floorsmacking_anim,
-        1.0f,
-        0.0f,
-        0.0f,
-        0,
-        -4.0f,
-    },
-    {
-        object_ginko_sitting_anim,
-        1.0f,
-        0.0f,
-        0.0f,
-        0,
-        -4.0f,
-    },
-    {
-        object_ginko_stamp_reach_anim,
-        1.0f,
-        0.0f,
-        0.0f,
-        2,
-        -4.0f,
-    },
-    {
-        object_ginko_advertising_anim,
-        1.0f,
-        0.0f,
-        0.0f,
-        0,
-        -4.0f,
-    }, // looking around for customers
-    {
-        object_ginko_amazed_anim,
-        1.0f,
-        0.0f,
-        0.0f,
-        0,
-        -4.0f,
-    },
+    { object_ginko_floorsmacking_anim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
+    { object_ginko_sitting_anim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
+    { object_ginko_stamp_reach_anim, 1.0f, 0.0f, 0.0f, 2, -4.0f },
+    { object_ginko_advertising_anim, 1.0f, 0.0f, 0.0f, 0, -4.0f }, // looking around for customers
+    { object_ginko_amazed_anim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
```

Comment:
```suggestion
    { objectGinkoFloorsmackingAnim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
    { objectGinkoSittingAnim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
    { objectGinkoStampReachAnim, 1.0f, 0.0f, 0.0f, 2, -4.0f },
    { objectGinkoAdvertisingAnim, 1.0f, 0.0f, 0.0f, 0, -4.0f }, // looking around for customers
    { objectGinkoAmazedAnim, 1.0f, 0.0f, 0.0f, 0, -4.0f },
```
Should these be changed to camel case?

Reply:
Those would need more changes to match the object's naming scheme we use. I think we may want to leave those renames to whoever decides to do this object in the future.